test/cases/installers.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
 set -euo pipefail
 
+#
+# tests the correct installation fo selected packaes. To do so:
+# Builds a tar-installer, installs the OS, boots it, and checks if the
+# packages selected in the blueprint are installed.
```

Comment:
Uhm, no... This test builds a `image-installer` iso which is then used for OS installation. The packages there are necessary for some additional checks I would assume. So what this test does is creates an `image-installer` iso, then modifies the kickstart so it can be installed automatically, installs the VM with `virt-install` waits for it to boot and then run some test using ansible to check if everything is as expected. Check the code and ask if you don't understand any parts.